macros/CompatBeta.ejs

```diff
@@ -105,9 +105,10 @@ function writeFeatureName(row, feature) {
   } else {
     desc += `<code>${Object.keys(row)[0]}</code>`;
   }
-  // Add relative MDN url, but don't add it for "Basic support"
-  if (feature.mdn_url && label != identifier) {
-    desc = `<a href="${feature.mdn_url.replace('https://developer.mozilla.org', '')}">${desc}</a>`;
+  // Add relative MDN url, but don't link to the current page
+  let href = feature.mdn_url.replace('https://developer.mozilla.org', '');
+  if (href && href != '/docs/' + env.slug) {
```

Comment:
Is the use of a single equals sign here intentional?

Reply:
No, good catch!